integration-tests/api/__tests__/store/orders.js

```diff
@@ -155,6 +155,64 @@ describe("/store/carts", () => {
       )
     })
 
+    it("lookup order response contains only fields defined with `fields` param", async () => {
+      const api = useApi()
+
+      const response = await api
+        .get(
+          "/store/orders?display_id=111&email=test@email.com&fields=status,object"
+        )
+        .catch((err) => {
+          return err.response
+        })
+
+      expect(Object.keys(response.data.order)).toEqual(
+        expect.arrayContaining([
```

Comment:
suggestion(non-blocking): Remove array pattern matching. Will make the `.length` check redundant 
```suggestion
```